conf/base.config

```diff
@@ -0,0 +1,42 @@
+/*
+ * -------------------------------------------------
+ *  nf-core/coproid Nextflow base config file
+ * -------------------------------------------------
+ * A 'blank slate' config file, appropriate for general
+ * use on most high performace compute environments.
+ * Assumes that all software is installed and available
+ * on the PATH. Runs in `local` mode - all jobs will be
+ * run on the logged in environment.
+ */
+
+process {
+  cpus = { check_max( 1 * task.attempt, 'cpus' )}
+  memory = { check_max( 1.GB * task.attempt, 'memory' )}
+  time = { check_max( 30.m * task.attempt, 'time' )}
+
+  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+  maxRetries = 1
+  maxErrors = '-1'
+
+  // Process-specific resource requirements
+  // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
+
+
+  withLabel:expresso{
+    cpus = { check_max(6 * task.attempt, 'cpus')}
+    memory = { check_max( 2.GB * task.attempt, 'memory' )}
+  }
+
+  withLabel:intenso{
+    cpus = { check_max(8 * task.attempt, 'cpus')}
+    memory = { check_max( 8.GB * task.attempt, 'memory' )}
+  }
+}
+
+params {
+  // Defaults only, expecting to be overwritten
+  max_memory = 128.GB
+  max_cpus = 16
+  max_time = 240.h
+  igenomes_base = 's3: //ngi-igenomes/igenomes/'
```

Comment:
There is a space in the `igenomes_base` path!

Reply:
Fixed, the format on save of VS-code tricked me ! ;)